Services/AccessControl/classes/class.ilConditionHandler.php

```diff
@@ -433,24 +434,24 @@ function enableAutomaticValidation($a_validate = true)
 	function getTriggerTypes()
 	{
 		global $objDefinition;
-		
-		$trigger_types =  array('crs','exc','tst','sahs', 'svy', 'lm', 'iass');
+
+		$trigger_types =  array('crs','exc','tst','sahs', 'svy', 'lm', 'iass', 'prg');
 
 		foreach($objDefinition->getPlugins() as $p_type => $p_info)
 		{
 			if(@include_once $p_info['location'].'/class.ilObj'.$p_info['class_name'].'Access.php')
 			{
 				include_once './Services/AccessControl/interfaces/interface.ilConditionHandling.php';
 				$name = 'ilObj'.$p_info['class_name'].'Access';
-				$refection = new ReflectionClass($name);
-				if($refection->implementsInterface('ilConditionHandling'))
+				$reflection = new ReflectionClass($name);
+				if($reflection->implementsInterface('ilConditionHandling'))
```

Comment:
👍  for removing spelling mistakes.

Please revert the whitespace changes, as @smeyer-ilias asked us.